TRB_label_assist/models.py

```diff
@@ -880,13 +887,19 @@ def set_params(self, params):
 
         return self
 
-    def fit(self, train_df):
+    def fit(self, train_df,ct_entry):
```

Comment:
`ct_entry` is the `Entry` type data kept aside when loading data and converting to initial dataframe. 

PS: `train_df` , the other parameter, is derived from  this initial dataframe.

Reply:
@humbleOldSage when I said 

> Some additional context in the commit message or as a comment would be helpful.

I meant "source code comments"
https://github.com/e-mission/e-mission-eval-private-data/pull/37#discussion_r1299412247

While this is better than nothing - it requires people to use git blame to find the commit, then find the related PR and scroll down to find these comments. And since we will be resolving them,  they would all be hidden, so they would not be searchable anyway.